convert.py

```diff
@@ -125,5 +124,9 @@ def crosscheck_existing():
 
 
 if __name__ == '__main__':
-    convert_2020()
+    # input_fp = 'localdata/NSIDC-0447-2020/cmc_daily_analysis_2020.txt'
+    # output_fp = 'localdata/convert_outputs_2021/Snow_Depth/Snow_Depth_Daily_Values/GeoTIFF/cmc_sdepth_dly_2020_v01.2.tif'
+    input_fp = 'localdata/cmc_sdepth_dly_2019_v01.2.txt'
+    output_fp = 'localdata/convert_outputs_2021/Snow_Depth/Snow_Depth_Daily_Values/GeoTIFF/cmc_sdepth_dly_2019_v01.3.tif'
+    convert_cmc_daily_analysis_txt_to_geotiff(input_fp, output_fp)
```

Comment:
TODO: put a check here that fails with an informative message if the `localdata/` dir does not exist.

This `__main__` block is really just for my personal use, so maybe just get rid of it and create a script that imports from this module!

Reply:
Done.